src/test/java/com/amazon/opendistro/elasticsearch/performanceanalyzer/rca/store/ResourceHeatMapGraphTest.java

```diff
@@ -603,10 +603,13 @@ private void verifyFullNodeTemperatureProfile(String resp) {
     JsonArray json = parser
         .parse(resp)
         .getAsJsonObject()
-        .getAsJsonArray(ALL_TEMPERATURE_DIMENSIONS)
-        .get(0)
-        .getAsJsonObject()
-        .getAsJsonArray(NodeLevelDimensionalSummary.SUMMARY_TABLE_NAME);
+        .getAsJsonArray(ALL_TEMPERATURE_DIMENSIONS);
+    // System.out.println("ALL_TEMPERATURE_DIMENSIONS" + json);
+
+    json.get(0)
+    .getAsJsonObject()
+    .getAsJsonArray(NodeLevelDimensionalSummary.SUMMARY_TABLE_NAME);
```

Comment:
return value is being ignored here. Do we need this change?

Reply:
ya. I was running tests and left it like that. good point !